src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -142,12 +142,17 @@ object DemuxFastqs {
   def demultiplexingIterator(sources: Seq[FastqSource],
                              demultiplexer: FastqDemultiplexer,
                              threads: Int,
+                             threshold: Int,
+                             qualityEncoding: QualityEncoding,
                              batchSize: Int = DemuxBatchRecordsSize): Iterator[DemuxResult] = {
 
     require(demultiplexer.expectedNumberOfReads == sources.length,
       s"The demultiplexer expects ${demultiplexer.expectedNumberOfReads} reads but ${sources.length} FASTQ sources given.")
 
     val zippedIterator = FastqSource.zipped(sources)
+
+    val thresh = qualityEncoding.toStandardAscii((threshold + qualityEncoding.asciiOffset).toChar)
```

Comment:
I like that you haven't pulled this out all the way to the `demuxFastqs` class, as this method is only called once anyhow.